page.codeberg.foreverxml.Random.json

```diff
@@ -0,0 +1,83 @@
+{
+    "app-id" : "page.codeberg.foreverxml.Random",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "randomgtk",
```

Comment:
You should also change the command name at https://codeberg.org/foreverxml/random/src/branch/main/data/page.codeberg.foreverxml.Random.desktop.in#L3. Currently it says `random` (not `randomgtk`), so desktop launcher doesn't work.

Reply:
I just changed this! Thanks, it slipped my mind.